docs/src/behavior.md

```diff
@@ -3,17 +3,19 @@ layout: two-columns
 title: Dropdown Behavior
 ---
 
-Our JS library `docsearch.js` is a wrapper of the [autocomplete.js][1] library.
-This library will listen to every keystrokes in the search input, query Algolia,
-and display the results in a dropdown. Everything is already configured for you
-to work with DocSearch. Our UI library also exposes configuration options you
-can use to go even further. You will discover Algolia.
+Our JS library `docsearch.js` is a wrapper of the [Algolia autocomplete.js][1]
+library. This library will listen to every keystrokes type in the search input,
+query Algolia, and display the results in a dropdown. Everything is already
+configured for you to work with DocSearch. Our UI library also exposes
+configuration options you can use to go even further. You will discover Algolia
+out of the box for documentation. Let's start the **learn as you type**
+experience.
 
```

Comment:
`You will discover Algolia out of the box for documentation. Let's start the **learn as you type** experience.`

I don't really understand these sentences, they sound weird. "You will discover" and "Let's start the XXX experience" sound like something coming out of a TV advert IMO, I'm not a fan

Reply:
I think it is fine to have our catchphrase here and explain we are providing Algolia out of the box